packages/ra-core/src/form/ValidationError.tsx

```diff
@@ -32,10 +32,10 @@ const ValidationError = (props: ValidationErrorProps) => {
             message,
             args,
         } = errorMessage as ValidationErrorMessageWithArgs;
-        return <>{translate(message, { _: message, ...args })}</>;
+        return <>{translate(message, args)}</>;
     }
 
-    return <>{translate(errorMessage as string, { _: errorMessage })}</>;
+    return <>{translate(errorMessage as string)}</>;
```

Comment:
Why did you remove that?

Reply:
To allow warnings on missing translations

Reply:
But then it's a regression, as missing translation warnings will now get logged in the console while they  were not in `master`. 

If this is only for the purpose of tests, you can enable warning on missing message in the parameters of the polyglotI18nProvider options (https://airbnb.io/polyglot.js/#options-overview). And if it doesn't work, you'll have to use a spy i18nProvider.

Reply:
The fact that it silenced the missing translations warnings was actually a reggresion and was contradictory to the expected and documented behavior.